ui/src/app/views/settings/component.tsx

```diff
@@ -61,8 +70,8 @@ export class SettingsReact extends React.Component<{}, SettingsState> {
       <h3>Notebooks</h3>
       <div style={styles.notebookSettings}>
         <TooltipTrigger content={
-                          (!this.state.cluster) ?
-                            'Your notebook server is still being created' : undefined}
+          this.state.waitingForUserProfile ? 'Waiting for notebook server status' :
```

Comment:
We haven't asked Leo about the cluster yet, so it might be ready - IMO it's inappropriate to claim that it's still being created

Reply:
+1 thanks for improving this